src/sentry/integrations/github/integration.py

```diff
@@ -291,8 +291,7 @@ def get_trees_for_org(self, cache_seconds: int = 3600 * 24) -> dict[str, RepoTre
 
         return trees
 
-    # TODO(cathy): define in issue ABC
-    def search_issues(self, query: str) -> Mapping[str, Sequence[Mapping[str, Any]]]:
+    def search_issues(self, query: str | None, **kwargs):
```

Comment:
Can we add the return type to this somehow? I'm assuming it's just a list of issues, or a serialized form of it?

Reply:
it's actually just `BaseApiResponseX` because all we do is call a client function 😬 
https://github.com/getsentry/sentry/blob/673e60633fcbedc08ed772aab29edc42e55fe6d7/src/sentry/shared_integrations/client/base.py#L30-L31

Reply:
i was able to narrow it down to a list of dicts or a dict. i don't think we're typing a lot of these places correctly (these are still in the allowlist in `pyproject.toml`) so i put that return type for now, we can fix it later